pkg/cli/admin/catalog/mirror.go

```diff
@@ -27,8 +27,8 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"github.com/alicebob/sqlittle"
-	"github.com/docker/distribution"
-	"github.com/docker/distribution/manifest/manifestlist"
+	"github.com/distribution/distribution/v3"
```

Comment:
Is this change necessary? I have a feeling this may break things

Reply:
nvm, this apparently comes from https://github.com/openshift/library-go/pull/1529/commits

Reply:
This was introduced recently in library-go: https://github.com/openshift/library-go/pull/1529

And unfortunately it's necessary wherever relevant library-go code is used :confused: